src/internal/ppsdb/ppsdb.proto

```diff
@@ -3,12 +3,16 @@ syntax = "proto3";
 package pps_v2;
 option go_package = "github.com/pachyderm/pachyderm/v2/src/internal/ppsdb";
 
+import "google/protobuf/timestamp.proto";
+
 message ClusterDefaultsWrapper {
   reserved 1, 2;
   reserved "details_json", "effective_details_json";
   string json = 3;
 }
 
 message ProjectDefaultsWrapper {
+  string created_by = 1;
```

Comment:
we used to have field 1 & 2 on this proto?

Reply:
Nope, `ProjectDefaultsWrapper` was a copy-paste of `ClusterDefaultsWrapper`, which _did_ have those fields, and maintains them as deprecated.